### PR TITLE
Feat/#21 weather info

### PR DESCRIPTION
**Pull request by McFly1208**
# Description

## What?

Added:
-  [x] Weather Info component

## Why?

-  Complete the second part of the Gps screen to obtain the weather. See #21  for more information.

## Testing

1. Install modules.

```
npm install
```

---

2. Then:

Test app:

```
npm test
```

Run the app:

```
npm start
```

## Notes:

- Design is not responsive.

## Screenshots

> If applicable, include screenshots of the results or screenshots that help to see changes

| Preview |
| :-----: |
|  image|
| image|

### CODE REVIEW

**Review by Manz1999 — APPROVED** [review]

**Review by garzo94 — APPROVED** [review]

**Review by HenrikhWolf — APPROVED** [review]

File: src/components/CWWidget.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class CWWidget extends Component<ICWWidgetProps> {
           />
         </View>
         <View style={this.props.style.rowContainer}>
-          <Text style={this.props.style.textDegrees}>{this.props.data.degrees}°</Text>
+          <Text style={this.props.style.textDegrees}>{this.props.data.degrees.toString()}°</Text>
```
Comment (yovanha) [inline src/components/CWWidget.tsx:27]: Please check the indentation of your files

Comment (arthurchina92) [reply src/components/CWWidget.tsx:27]: Mmm I think it's correctly indented, we are using Prettier and ESLint, and we don't see any issues, also Codacy didn't drop any alert.
Preview (no issues):
image

Preview (with issues):
image

Codacy results: https://app.codacy.com/gh/BrightCoders-Institute/BCDIC22-RN-proyecto-final-team3/pullRequest?prid=11325051&bid=34206334.

Comment (yovanha) [reply src/components/CWWidget.tsx:27]: I see, in that case it looks good to me

**Review by yovanha — APPROVED** [review]

File: src/components/CWWidget.tsx (hunk 1 above)
Comment (arthurchina92) [reply src/components/CWWidget.tsx:27]: Thanks

**Pull request merged**